### ISSUE TITLE
with settings the blur kernel size 8x8 or 32x32 and same as stride, it doesnt work well

### ISSUE THREAD

**Issue by TIMESTICKING** [JiangtaoNie/DBSR#1]
dear author, i sent Lei Zhang an email abt the customized settings that is when i set kernel size of 8x8 and same as stride, or 32x32 kernel size and same as stride, all the other settings are default, then the psnr i got is terrible

im wondering which part needs to change that i missed, and would u provide ur degraded LR-HSI and HR-MSI plz. Thks alot

**Comment by JiangtaoNie** [comment]
Have you ever nomalized the distribution of input into [0, 1], if not, this may caused worse performance.

**Comment by TIMESTICKING** [comment]
> Have you ever nomalized the distribution of input into [0, 1], if not, this may caused worse performance.

thanks for replying. yes I did, and actually 
`im_gt = im_gt/(torch.max(im_gt)-torch.min(im_gt))`
it's ur code.
I only change the inputs, like gt, lr-hsi, hr-msi, and how I generated them is like I said above. Im so confused.

**Comment by JiangtaoNie** [comment]
I don`t understand what kind of method you used to generate the low-resolution HSI. In ours work,  we utilize a convolution layer to apply the operation of downsampling, conv kernel is substituted by a blur kernel and the stide is set same as the downsample scale. Not same as the size as kernel size.

**Comment by TIMESTICKING** [comment]
> I don`t understand what kind of method you used to generate the low-resolution HSI. In ours work, we utilize a convolution layer to apply the operation of downsampling, conv kernel is substituted by a blur kernel and the stide is set same as the downsample scale. Not same as the size as kernel size.

yeah, me the same. The only difference is the downsample scale is set to equal to kernel size, since it's different from yours, what else hyper-parameters should I also change.

**Comment by Jessejx** [comment]
> yeah, me the same. The only difference is the downsample scale is set to equal to kernel size, since it's different from yours, what else hyper-parameters should I also change.

i got terrible psnr  too. Have u solved this problem?

**Comment by TIMESTICKING** [comment]
> 

It's been ages, and my memory got blurred, I don't remember what I changed, but It works fine now. 

I think it's about the pre-train part that I modified, the `scale_factor` in the function `Spa_UpNet` should be modified accordingly

**Comment by Jessejx** [comment]
> 

Thanks for your suggestion, I am going to modify the pretraining part immediately.  IMeantime, if you require any further information, please let me know

**Comment by Jessejx** [comment]
> > 
> 
> It's been ages, and my memory got blurred, I don't remember what I changed, but It works fine now.
> 
> I think it's about the pre-train part that I modified, the `scale_factor` in the function `Spa_UpNet` should be modified accordingly

Thanks for your suggestion, I am going to modify the pretraining part immediately. IMeantime, if you require any further information, please let me know

**Comment by Jessejx** [comment]
> > 
> 
> It's been ages, and my memory got blurred, I don't remember what I changed, but It works fine now.
> 
> I think it's about the pre-train part that I modified, the `scale_factor` in the function `Spa_UpNet` should be modified accordingly

Thanks for replying
I have modified the pretraining part parameters, but it seems doesn't work. Would u send your demo to this email: [ljx9339@hotmail.com](mailto:ljx9339@hotmail.com)?
